calico/getting-started/kubernetes/managed-public-cloud/aks.mdx

```diff
@@ -9,15 +9,62 @@ title: Installing on AKS
+Azure Kubernetes Service (AKS) supports multiple configurations for networking and network policy using Calico.
+Your choice determines how networking is implemented, who manages Calico, and which features are available.
 
-You can also use $[prodname] for networking on AKS in place of the default Azure VPC networking. This allows you to take advantage of the full set of $[prodname] networking features.
+### Option 1: Azure-managed Calico (built-in)
+
+With this option, AKS uses the Azure CNI plugin for networking and the Azure-managed Calico implementation for network policy.
+
+- **Networking:** Provided by Azure CNI
+- **Network policy:** Implemented by Azure-managed Calico
+- **Management:** Calico components are installed, upgraded, and supported by Azure
+- **Features:** Full Kubernetes Network Policy API (standard features only)
+
+### Option 2: Self-managed Calico for network policy
+
+In this setup, your cluster still uses Azure CNI for networking, but you install and manage Calico yourself to enable advanced policy and observability capabilities.
+
+- **Networking:** Provided by Azure CNI
+- **Network policy:** Implemented by self-managed Calico
+- **Management:** You install, configure, and upgrade Calico
+- **Support:**
+  - Cluster supported by Azure
+  - Calico features supported by the community or a commercial support plan
+- **Features:**
+  - Advanced Calico policy and global network policy
+  - Whisker observability and other enterprise-grade capabilities
+
+This approach gives you more control and access to advanced Calico policy features while still benefiting from Azure’s managed networking infrastructure.
+
+### Option 3: Self-managed Calico for networking and network policy
+
+You can also use Calico CNI as the primary networking plugin in AKS, replacing Azure CNI entirely.
+This configuration provides the full Calico networking and policy feature set.
+
+- **Networking:** Provided by Calico CNI
+- **Network policy:** Implemented by Calico
+- **Management:** You install, configure, and upgrade Calico
+- **Support:**
+  - Cluster supported by Azure
+  - Calico features supported by the community or a commercial support plan
+- **Features:**
+  - Full Calico networking stack
+  - Advanced policy, global policy, and observability tools
+
+This option gives you maximum flexibility and functionality, but requires you to manage Calico yourself.
+
+## Prerequisites
+
+- You have an Azure account with permissions to create resource groups and AKS clusters
+- You installed and configured the [az command line tool](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli).
+- You have [logged into Azure](https://docs.microsoft.com/en-us/cli/azure/authenticate-azure-cli) and [created a service principal](https://docs.microsoft.com/en-us/cli/azure/create-an-azure-service-principal-azure-cli)
```

Comment:
Can we do without the logged in part? Is it really a requirement for any of the following procedures? (If it's a requirement for just the service principal thing, that doesn't count for our purposes. They'll figure that out when they do that with the Microsoft docs).